sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -55,6 +64,9 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
         });
 
         this.itemViews = itemCards;
+        if (this.itemViews.length > 0) {
```

Comment:
If we only update change "unread" when an item is read in the collection, then do we need to also do this in the render at all?